### ISSUE TITLE
Miss nginx/apache2 configuration with specific domain and broke vestacp menu

### ISSUE THREAD

**Issue by franciscopaniskaseker** [serghey-rodin/vesta#1324]
### Operating System (OS/VERSION):

CentOS 7

### VestaCP Version:

root@ax:~# dpkg -l | egrep -i vesta
ii  vesta                             0.9.8-17                                   amd64        Vesta
ii  vesta-nginx                       0.9.8-17                                   amd64        Vesta Nginx
ii  vesta-php                         0.9.8-17                                   amd64        Vesta php-fpm

### Installed Software (what you got with the installer):

nginx, php-fpm, apache, php

### Steps to Reproduce:

(1) Add new domain
(2) Write "equipamentocervejeiro.com.br"
(3) Try to add

### Related Issues/Forum Threads:

Only with that domain the problem occurred. All configurations in nginx.conf get a wrong "tab".

example:
```bash
# what is configured by vestacp
alias   /home/admin/web/equipamentocervejeiro.com.br    /document_errors/;
include /home/admin/conf/web/nginx.equipamentocervejeiro.com.br     .conf*;
root           /home/admin/web/equipamentocervejeiro.com.br /public_html;
error_log  /var/log/apache2/domains/equipamentocervejeiro.com.br        .error.log error;
access_log     /var/log/apache2/domains/equipamentocervejeiro.com.br        .log combined;
access_log     /var/log/apache2/domains/equipamentocervejeiro.com.br        .bytes bytes;
# what would be
alias   /home/admin/web/equipamentocervejeiro.com.br    /document_errors/;
include /home/admin/conf/web/nginx.equipamentocervejeiro.com.br     .conf*;

root           /home/admin/web/equipamentocervejeiro.com.br/public_html;
error_log  /var/log/apache2/domains/equipamentocervejeiro.com.br.error.log error;
access_log     /var/log/apache2/domains/equipamentocervejeiro.com.br.log combined;
access_log     /var/log/apache2/domains/equipamentocervejeiro.com.br.bytes bytes;
```

The same thing in apache2 configuration (IncludeOptional, documentroot etc).

No idea why vestacp put "tabs". I have to fix manually.

After fixed every apache2 and ngix configuration, "WEB" tab from vestacp does not list any domain (but the menu can count how much domains and aliases). Other tabs work normally.

I saw that problem (list web domains) only occur at user that have this problem. I logged in other account and no issues.

So I tried:

```bash
v-rebuild-web-domains admin
```

with no success. 

Tried to rebuild user admin (v-rebuild-user) and nothing. And when I tried do logout and login I lost my access.

Then I restored last backup and I even after that I can not login. So I executed "v-change-user-password" to change password and I continue getting error to do login.

In auth.log when I try to do ONE login pam print:

```bash
Nov 13 13:49:14 ax sudo: pam_unix(sudo:session): session opened for user root by (uid=0)
Nov 13 13:49:14 ax sudo: pam_unix(sudo:session): session closed for user root
Nov 13 13:49:14 ax sudo: pam_unix(sudo:session): session opened for user root by (uid=0)
Nov 13 13:49:14 ax sudo: pam_unix(sudo:session): session closed for user root
```

I tried restart vestacp and nothing... then I said f*** off and rebooted the server... now I can login, but web tab continue not listing anything... and others users are ok.

I decided to click in "+" in WEB tab to add a new domain and I get "internal vestacp error with moon logo.  Before that I can not see any error from /var/log/vesta/. When I get "internal vestacp error" clicking in "+" button, I saw one error from "/var/log/vesta/nginx-error.log":

```bash
2017/11/13 13:55:01 [error] 1530#0: *20 "/usr/local/vesta/web/add/web/index.php" is forbidden (13: Permission denied), client: 179.154.53.184, server: _, request: "GET /add/web/ HTTP/1.1", host: "ilax.com.br:8083", referrer: "https://ilax.com.br:8083/list/web/"
```

I do not know what is correct permissions for that file.

```bash
root@ax:~# ls -l /usr/local/vesta/web/add/web/index.php
-rw-r--r-- 1 root root 15976 Nov 25  2016 /usr/local/vesta/web/add/web/index.php
```

From now I do not know what I can do to fix that...

**Comment by franciscopaniskaseker** [comment]
At /usr/local/vesta/data/users/admin/web.conf file I saw a lot of "tab" spaces on that domain.

I fixed that and now vestacp can list again my domains.

So the main question is: Why vestacp put a lot of tabs?

**Comment by dpeca** [comment]
Can you reproduce bug again?
I mean, delete domain andc try to add domain again?
Just to be sure you didnt't (at first time) copied domain with tab character...

**Comment by noogen** [comment]
I'm guessing, when you enter the domain name on the UI, it has tab/spaces? like so "equipamentocervejeiro.com.br     "

That would explain all those extra tabs.   I just tried this myself and broke my vesta.  I added a website with a bunch of tabs "dummy.com					" from the UI.  I guess the bug fix is that the UI need to strip/remove all spaces before it use the domain.

**Comment by franciscopaniskaseker** [comment]
Yes, I can reproduce. If you send with tab, it will bug a lot of files.
